src/app/pages/applications/forms/chart-release-edit.component.ts

```diff
@@ -411,6 +424,9 @@ export class ChartReleaseEditComponent {
     data.config.pullPolicy = data.config.image.pullPolicy;
     data.config.nameservers = data.config.dnsConfig.nameservers;
     data.config.searches = data.config.dnsConfig.searches;
+    if (data.config.securityContext && data.config.securityContext) {
```

Comment:
Two statements in if condition are identical.

Reply:
@undsoft Fixed